### PR TITLE
Added performance enhancement.

### PR DESCRIPTION
**Pull request by thomasquintana**

### CODE REVIEW

**Comment by lylepratt** [comment]
Looks great!

**Comment by lylepratt** [comment]
More threads!

**Comment by lylepratt** [comment]
@thomasquintana approved

File: freepy/lib/server.py [hunk 1]
```diff
@@ -96,27 +97,37 @@ def start(self, wait_for_signal=True):
...
+        while True:
+          if not scheduler.is_running:
+            break  
       reactor.stop()
     signal.signal(signal.SIGINT, signal_handler)
     # Start the scheduler thread.
-    scheduler_thread = Thread(target=scheduler.start)
-    scheduler_thread.daemon = True
-    scheduler_thread.start()
+    for scheduler in schedulers:
```
Comment (thomasquintana) [inline freepy/lib/server.py]: Yeah this should def get us a big improvement.

**Pull request merged**